login.php

```diff
@@ -1,3 +1,30 @@
+<?php
+	ob_start();
```

Comment:
This code here is for the user login and ensures that only correct login details can be used to access the website